c2rust-transpile/src/translator/simd.rs

```diff
@@ -100,6 +102,21 @@ impl<'c> Translation<'c> {
                     ).into());
                 }
 
+                // Drop the `_u` suffix for unaligned SIMD types as they have
+                // no equivalents in Rust. Warn the user about possible probems.
```

Comment:
Can we use this for unaligned SIMD types?

[Godbolt](https://godbolt.org/z/PhY635j4d)

```rust
use core::arch::x86_64::__m256;
use core::mem;

#[unsafe(no_mangle)]
pub fn aligned() -> usize {
    mem::align_of::<__m256>()
}

#[repr(packed)]
pub struct UnalignedM256(pub __m256);

#[unsafe(no_mangle)]
pub fn unaligned() -> usize {
    mem::align_of::<UnalignedM256>()
}
```

`#[repr(packed)]` lets you override the alignment and unalign a type.